packages/dcos-launch/extra/README.md

```diff
@@ -0,0 +1,28 @@
+# dcos-launch User Guide
+## Purpose
+`dcos-launch` is intendends to provide the following:
+* Consistent Interface: launching on Azure or AWS can involve finding a suitable client API and then sifting through hundreds of specific API calls to find the combination that will allow deploying DC/OS. `dcos-launch` has the exact methods required to take a DC/OS build artifact and deploy it to an arbitrary provider with no other tools.
+* Turn-key Deployment: The only input required by dcos-launch is the [launch config](launch/sample_configs/README.md). In cases where artifacts are required to deploy DC/OS (e.g. SSH keys), `dcos-launch` provids helpers to automatically create and cleanup those extra artifacts
+* Portable shipping: `dcos-launch` is shipped as a frozen python binary so that you never need to be concerned with maintaining an extremely specific develop environment or deployment framework.
+* Build Verification: besides launching clusters, `dcos-launch` can test that the provided cluster is healthy through the `pytest` subcommand. `dcos-launch` is capable of describing the cluster topology and can verify that the expected cluster is present. To learn more about the testing suite triggered by this command, see: [dcos-integration-tests](../../dcos-integration-tests/extra)
+* Programmatic consumption: all information necessary to handle a given deployment created by `dcos-launch` is contained in the generated `cluster_info.json` file. This allows dependent CI tasks to run other tests, describe the cluster, or completely delete the cluster
```

Comment:
Some of the headings in the above list capitalize each word, while others only capitalize the first word. I prefer the latter format, but I'll leave that up to you; it should at least be consistent.